servo/types/slo.py

```diff
@@ -42,6 +44,19 @@ def _check_threshold_values(cls, values):
 
         return values
 
+    @pydantic.root_validator
+    @classmethod
+    def _check_duplicated_minimum(cls, values):
+        if (
+            values.get("threshold") is not None
+            and values.get("slo_threshold_minimum") is not None
+        ):
+            raise ValueError(
+                "SLO Condition cannot specify both static threshold and metric based threshold minimum"
+            )
```

Comment:
Also try and include some sort of identifier for the condition in the warning message
```suggestion
            # Use run time import to prevent circular imports
            import servo.logging
            servo.logging.logger.warning(
                "SLO Condition should not specify both static threshold and metric based threshold minimum."
                " Please double check the Slo Conditions of the User Config"
            )
```